projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -4447,6 +4452,23 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         return pagingHeight;
     }
 
+    /**
+     * @hidden
+     */
+    protected getFilterRowHeight(): number {
+        let filterRowHeight = 0;
+        if (this.filteringRow) {
```

Comment:
You could remove the filteringRow check and always get the filtering cell height.

Reply:
Done